app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+
+let Comment = React.createClass({
+  propTypes: {
+    comment: React.PropTypes.object,
```

Comment:
Try to be as specific as possible regarding object types. I recommend using `React.PropTypes.shape` so we can define object keys and their types.